### PR TITLE
Fix default value for tool_objectfs/expirationtime setting 

### PR DESCRIPTION
**Pull request by golenkovm**
Fix for #233

### CODE REVIEW

File: settings.php [hunk 1]
```diff
@@ -102,7 +102,7 @@
 
         $settings->add(new admin_setting_configduration('tool_objectfs/expirationtime',
             new lang_string('settings:presignedurl:expirationtime', 'tool_objectfs'),
-            new lang_string('settings:presignedurl:expirationtime_help', 'tool_objectfs'), '', 10 * MINSECS));
+            new lang_string('settings:presignedurl:expirationtime_help', 'tool_objectfs'), 10, MINSECS));
```
Comment (gleimermora) [inline settings.php]: I think it should be `10 * MINSECS` if we want `10 mins`

Comment (brendanheywood) [reply settings.php]: This should be minutes, and also 10 is waaaaayyy to short. This needs to be around the same as the session expiry.

$CFG->sessiontimeout now defaults to 8 hours, so lets make this expiry at least 2 hours

File: settings.php [hunk 2]
```diff
@@ -102,7 +102,7 @@
 
         $settings->add(new admin_setting_configduration('tool_objectfs/expirationtime',
             new lang_string('settings:presignedurl:expirationtime', 'tool_objectfs'),
-            new lang_string('settings:presignedurl:expirationtime_help', 'tool_objectfs'), '', 10 * MINSECS));
+            new lang_string('settings:presignedurl:expirationtime_help', 'tool_objectfs'), 2 * HOURSECS, MINSECS));
```
Comment (golenkovm) [inline settings.php]: Let's set $defaultunit to HOURSECS as well

Comment (gleimermora) [reply settings.php]: updated

**Pull request merged**